src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -647,22 +647,19 @@ public void SetMembrane(string membraneName)
         gui.UpdateMembraneButtons(Membrane.InternalName);
     }
 
-    public void SetRigidity(float rigidity)
+    public void SetRigidity(int rigidity)
     {
-        if (Math.Abs(Rigidity - rigidity) < MathUtils.EPSILON)
+        if (Math.Abs(Rigidity * 10 - rigidity) < MathUtils.EPSILON * 10)
```

Comment:
I'd prefer the opposite check. ie. convert Rigidity to int and then compare if the ints are the same.
Also multiplying the EPSILON is not good, it's meant to be a value close to zero to check if a float is basically.